WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/HoloLens/PerceptionSimulationPlayback.cs

```diff
@@ -49,15 +55,183 @@ public partial class DevicePortal
         /// <summary>
         /// API for starting playback of a Holographic Perception Simulation recording.
         /// </summary>
-        public static readonly string HolographicSimulationStopApi = "api/holographic/simulation/playback/session/stop";
+        public static readonly string HolographicSimulationPlaybackStopApi = "api/holographic/simulation/playback/session/stop";
 
         /// <summary>
         /// API for retrieving the types of data in a Holographic Perception Simulation recording.
         /// </summary>
-        public static readonly string HolographicSimulationTypesApi = "api/holographic/simulation/playback/session/types";
+        public static readonly string HolographicSimulationPlaybackDataTypesApi = "api/holographic/simulation/playback/session/types";
+
+        /// <summary>
+        /// Deletes the specified Holographic Simulation recording.
+        /// </summary>
+        /// <param name="name">The name of the recording to delete (ex: testsession.xef).</param>
+        /// <remarks>This method is only supported on HoloLens devices.</remarks>
+        public async Task DeleteHolographicSimulationRecording(string name)
```

Comment:
Even those these don't really return anything, StyleCop wants a \<return\> param since they technically return a Task. See the other methods I cleaned up for a good copy/paste (something like "Returns a task tracking completion of the API").